src/Tribe/Main.php

```diff
@@ -4610,5 +4557,225 @@ public function set_meta_factory_global() {
+				}
+				if ( isset( $tec_info['num_ratings'] ) ) {
+					$num_rating = $tec_info['num_ratings'];
+				}
+				if ( isset( $tec_info['requires'] ) ) {
+					$requires = $tec_info['requires'];
+				}
+				if ( isset( $tec_info['version'] ) ) {
+					$version = $tec_info['version'];
+				}
+				$total_downloads = ( isset( $tec_info['total_downloads'] ) ) ? number_format( $tec_info['total_downloads'] ) : _x( 'n/a', 'not available', 'tribe-common' );
+				$up_to_date      = ( isset( $tec_info['version'] ) && version_compare( Tribe__Events__Main::VERSION, $tec_info['version'], '<' ) ) ? __( 'You need to upgrade!', 'tribe-common' ) : __( 'You are up to date!', 'tribe-common' );
+			}
+
+			?>
+			<div id="tribe-help-plugin-info">
+				<h3><?php esc_html_e( 'The Events Calendar', 'tribe-common' ); ?></h3>
+
+
+				<?php if ( isset( $up_to_date ) ) { ?><p><?php echo $up_to_date; ?></p><?php } ?>
+				<?php if ( isset( $version ) ) { ?><p>
+					<b><?php esc_html_e( 'Latest Version:', 'tribe-common' ); ?></b> <?php echo $version; ?>
+					<br /><?php } ?>
+					<b><?php esc_html_e( 'Author:', 'tribe-common' ); ?></b> <?php esc_html_e( 'Modern Tribe Inc', 'tribe-common' ); ?>
+					<br />
+					<?php
+					if ( isset( $requires ) ) {
+						?>
+						<b><?php esc_html_e( 'Requires:', 'tribe-common' ); ?></b> <?php esc_html_e( 'WordPress ', 'tribe-common' );
+						echo $requires; ?>+<br />
+						<?php
+					}
+					/**
+					 * Filter the URL to The Events Calendar plugin page on Wordpress.org
+					 *
+					 * @param string $url
+					 */
+					$tribe_help_tab_wp_plugin_url = apply_filters( 'tribe_help_tab_wp_plugin_url', Tribe__Events__Main::WP_PLUGIN_URL );
+					?>
+					<a href="<?php echo esc_url( $tribe_help_tab_wp_plugin_url ); ?>"><?php esc_html_e( 'Wordpress.org Plugin Page', 'tribe-common' ); ?></a>
+				</p>
+			</div>
+
+			<?php
+			if ( isset( $rating ) && isset( $num_rating ) ) {
+				?>
+				<h3><?php esc_html_e( 'Average Rating', 'tribe-common' ); ?></h3>
+				<?php wp_star_rating( array(
+					'rating' => $rating,
+					'type'   => 'percent',
+					'number' => $num_rating,
+				) ); ?>
+				<?php printf( _n( 'Based on %d rating', 'Based on %d ratings', $num_rating, 'tribe-common' ), $num_rating ); ?>
+				<p>
+					<?php
+					/**
+					 * Filter the URL to The Events Calendar plugin page on Wordpress.org
+					 *
+					 * @param string $url
+					 */
+					$tribe_help_tab_wp_plugin_url = apply_filters( 'tribe_help_tab_wp_plugin_url', 'http://wordpress.org/support/view/plugin-reviews/the-events-calendar?filter=5' );
+					?>
+					<a href="<?php echo esc_url( $tribe_help_tab_wp_plugin_url ); ?>"><?php esc_html_e( 'Give us 5 stars!', 'tribe-common' ); ?></a>
+				</p>
+				<?php
+			}
+		}
+
+		/**
+		 * Adds post types to the post_types array used to determine if on a post type screen
+		 *
+		 * @param array $post_types Collection of post types
+		 */
+		public function is_post_type_screen_post_types( $post_types ) {
```

Comment:
I realize the method name is just echoing the name of the matching hook - but might something like `add_event_post_types` be clearer?